it_zwierzak/adoption_announcement/models.py

```diff
@@ -1,28 +1,54 @@
+from django.conf import settings
 from django.db import models
+from django.core.validators import MaxValueValidator, MinValueValidator
 
 
 class Announcement(models.Model):
-    gender = models.CharField(max_length=10)
-    city = models.CharField(max_length=30)
-    color = models.CharField(max_length=20)
-    age = models.SmallIntegerField()
-    species = models.CharField(max_length=30)
-    breed = models.CharField(max_length=30)
-    name = models.CharField(max_length=15)
+    Animal = models.ForeignKey('Animal', on_delete=models.CASCADE,default=None)
+    Animal_Bio = models.TextField(max_length=3000, default=' ')
 
-    health = models.TextField()
-    description = models.TextField()
+    post_date = models.DateTimeField(auto_now_add=True)
+    added_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        limit_choices_to={'is_staff': True}, null=True
+    )
 
     def __str__(self):
-        return f'{self.species} {self.name} - {self.age} lat'
+        return f'{self.Animal}'
```

Comment:
dajmy coś więcej, bo to będzie to samo się wyświetlać co dla str(Animal), a jednak wolałbym widzieć, że to jest ogłoszenie